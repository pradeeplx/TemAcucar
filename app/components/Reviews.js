import React, { Component } from 'react'
import { Platform, View, Text, ActivityIndicator } from 'react-native'

import Colors from "../Colors"
import Sentence from "./Sentence"
import LoadMore from "./LoadMore"
import Review from "./Review"
import NoReviews from "./NoReviews"

export default class Reviews extends Component {
  render() {
    const { onList, user, reviews, listing, canList, currentUser } = this.props
    return (
      <View style={{
        flex: 1,
        flexDirection: 'column',
        alignSelf: 'stretch',
        alignItems: 'center',
      }}>
        { reviews.map((review, index) => (
          <Review
            key={review.id}
            review={review}
            index={index}
            currentUser={currentUser}
          />
        )) }
        { listing && <ActivityIndicator style={{ marginTop: 20 }} /> }
        { reviews.length === 0 && !listing && <NoReviews user={user} currentUser={currentUser} /> }
        { canList && !listing &&
          <LoadMore onPress={onList} style={{
            marginTop: 20,
          }} />
        }
      </View>
    )
  }
}
