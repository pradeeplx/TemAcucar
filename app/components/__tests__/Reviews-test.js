import 'react-native'
import React from 'react'
import Reviews from '../Reviews'
import renderer from 'react-test-renderer'

const props = {
  user: { id: 1 },
  reviews: [{
    id: 1,
    reviewer: {
      first_name: 'foo',
      last_name: 'bar',
      image_url: '#',
    },
  }, {
    id: 2,
    reviewer: {
      first_name: 'foo2',
      last_name: 'bar2',
      image_url: '#',
    },
  }],
  currentUser: { id: 1 },
}

it('renders correctly', () => {
  const tree = renderer.create(
    <Reviews {...props} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders ActivityIndicator', () => {
  const tree = renderer.create(
    <Reviews {...props} listing />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders load more button', () => {
  const tree = renderer.create(
    <Reviews {...props} canList />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders NoReviews when no review is given', () => {
  const tree = renderer.create(
    <Reviews {...props} reviews={[]} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('does not render NoReviews when listing', () => {
  const tree = renderer.create(
    <Reviews {...props} reviews={[]} listing />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
