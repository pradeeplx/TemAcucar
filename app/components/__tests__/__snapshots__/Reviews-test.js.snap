exports[`test does not render NoReviews when listing 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "flex": 1,
      "flexDirection": "column",
    }
  }>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="small"
    style={
      Object {
        "marginTop": 20,
      }
    } />
</View>
`;

exports[`test renders ActivityIndicator 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "flex": 1,
      "flexDirection": "column",
    }
  }>
  <View
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#FFFFFF",
        "borderBottomWidth": 12,
        "borderColor": "#FAF6F1",
        "flex": 1,
        "flexDirection": "row",
        "paddingVertical": 10,
      }
    }>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 2,
          "flexDirection": "column",
        }
      }>
      <Image
        source={
          Object {
            "uri": "#",
          }
        }
        style={
          Array [
            Object {
              "borderRadius": 27,
              "height": 54,
              "width": 54,
            },
            undefined,
          ]
        } />
    </View>
    <View
      style={
        Object {
          "flex": 8,
          "flexDirection": "column",
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Array [
              Object {
                "color": "#44B5B1",
                "fontSize": 18,
              },
              Object {
                "fontFamily": "Avenir-Black",
              },
            ],
          ]
        }>
        foo bar
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Array [
              Object {
                "fontSize": 13.5,
              },
              Object {
                "color": "#000000",
                "fontSize": 18,
              },
            ],
          ]
        }>
        Há segundos
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Object {
              "color": "#807F7F",
              "fontSize": 15,
              "marginBottom": 6,
            },
          ]
        }>
        foo não descreveu sua experiência.
      </Text>
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginBottom": 4,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#FFFFFF",
        "borderBottomWidth": 12,
        "borderColor": "#FAF6F1",
        "flex": 1,
        "flexDirection": "row",
        "paddingVertical": 10,
      }
    }>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 2,
          "flexDirection": "column",
        }
      }>
      <Image
        source={
          Object {
            "uri": "#",
          }
        }
        style={
          Array [
            Object {
              "borderRadius": 27,
              "height": 54,
              "width": 54,
            },
            undefined,
          ]
        } />
    </View>
    <View
      style={
        Object {
          "flex": 8,
          "flexDirection": "column",
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Array [
              Object {
                "color": "#44B5B1",
                "fontSize": 18,
              },
              Object {
                "fontFamily": "Avenir-Black",
              },
            ],
          ]
        }>
        foo2 bar2
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Array [
              Object {
                "fontSize": 13.5,
              },
              Object {
                "color": "#000000",
                "fontSize": 18,
              },
            ],
          ]
        }>
        Há segundos
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Object {
              "color": "#807F7F",
              "fontSize": 15,
              "marginBottom": 6,
            },
          ]
        }>
        foo2 não descreveu sua experiência.
      </Text>
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginBottom": 4,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
      </View>
    </View>
  </View>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="small"
    style={
      Object {
        "marginTop": 20,
      }
    } />
</View>
`;

exports[`test renders NoReviews when no review is given 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "flex": 1,
      "flexDirection": "column",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "#654C50",
          "fontFamily": "Avenir",
          "fontSize": 18,
        },
        Object {
          "alignSelf": "stretch",
          "fontSize": 15,
          "margin": 10,
          "textAlign": "center",
        },
      ]
    }>
    Você
     ainda não recebeu nenhuma avaliação.
  </Text>
</View>
`;

exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "flex": 1,
      "flexDirection": "column",
    }
  }>
  <View
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#FFFFFF",
        "borderBottomWidth": 12,
        "borderColor": "#FAF6F1",
        "flex": 1,
        "flexDirection": "row",
        "paddingVertical": 10,
      }
    }>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 2,
          "flexDirection": "column",
        }
      }>
      <Image
        source={
          Object {
            "uri": "#",
          }
        }
        style={
          Array [
            Object {
              "borderRadius": 27,
              "height": 54,
              "width": 54,
            },
            undefined,
          ]
        } />
    </View>
    <View
      style={
        Object {
          "flex": 8,
          "flexDirection": "column",
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Array [
              Object {
                "color": "#44B5B1",
                "fontSize": 18,
              },
              Object {
                "fontFamily": "Avenir-Black",
              },
            ],
          ]
        }>
        foo bar
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Array [
              Object {
                "fontSize": 13.5,
              },
              Object {
                "color": "#000000",
                "fontSize": 18,
              },
            ],
          ]
        }>
        Há segundos
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Object {
              "color": "#807F7F",
              "fontSize": 15,
              "marginBottom": 6,
            },
          ]
        }>
        foo não descreveu sua experiência.
      </Text>
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginBottom": 4,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#FFFFFF",
        "borderBottomWidth": 12,
        "borderColor": "#FAF6F1",
        "flex": 1,
        "flexDirection": "row",
        "paddingVertical": 10,
      }
    }>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 2,
          "flexDirection": "column",
        }
      }>
      <Image
        source={
          Object {
            "uri": "#",
          }
        }
        style={
          Array [
            Object {
              "borderRadius": 27,
              "height": 54,
              "width": 54,
            },
            undefined,
          ]
        } />
    </View>
    <View
      style={
        Object {
          "flex": 8,
          "flexDirection": "column",
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Array [
              Object {
                "color": "#44B5B1",
                "fontSize": 18,
              },
              Object {
                "fontFamily": "Avenir-Black",
              },
            ],
          ]
        }>
        foo2 bar2
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Array [
              Object {
                "fontSize": 13.5,
              },
              Object {
                "color": "#000000",
                "fontSize": 18,
              },
            ],
          ]
        }>
        Há segundos
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Object {
              "color": "#807F7F",
              "fontSize": 15,
              "marginBottom": 6,
            },
          ]
        }>
        foo2 não descreveu sua experiência.
      </Text>
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginBottom": 4,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`test renders load more button 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "stretch",
      "flex": 1,
      "flexDirection": "column",
    }
  }>
  <View
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#FFFFFF",
        "borderBottomWidth": 12,
        "borderColor": "#FAF6F1",
        "flex": 1,
        "flexDirection": "row",
        "paddingVertical": 10,
      }
    }>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 2,
          "flexDirection": "column",
        }
      }>
      <Image
        source={
          Object {
            "uri": "#",
          }
        }
        style={
          Array [
            Object {
              "borderRadius": 27,
              "height": 54,
              "width": 54,
            },
            undefined,
          ]
        } />
    </View>
    <View
      style={
        Object {
          "flex": 8,
          "flexDirection": "column",
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Array [
              Object {
                "color": "#44B5B1",
                "fontSize": 18,
              },
              Object {
                "fontFamily": "Avenir-Black",
              },
            ],
          ]
        }>
        foo bar
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Array [
              Object {
                "fontSize": 13.5,
              },
              Object {
                "color": "#000000",
                "fontSize": 18,
              },
            ],
          ]
        }>
        Há segundos
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Object {
              "color": "#807F7F",
              "fontSize": 15,
              "marginBottom": 6,
            },
          ]
        }>
        foo não descreveu sua experiência.
      </Text>
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginBottom": 4,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#FFFFFF",
        "borderBottomWidth": 12,
        "borderColor": "#FAF6F1",
        "flex": 1,
        "flexDirection": "row",
        "paddingVertical": 10,
      }
    }>
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 2,
          "flexDirection": "column",
        }
      }>
      <Image
        source={
          Object {
            "uri": "#",
          }
        }
        style={
          Array [
            Object {
              "borderRadius": 27,
              "height": 54,
              "width": 54,
            },
            undefined,
          ]
        } />
    </View>
    <View
      style={
        Object {
          "flex": 8,
          "flexDirection": "column",
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Array [
              Object {
                "color": "#44B5B1",
                "fontSize": 18,
              },
              Object {
                "fontFamily": "Avenir-Black",
              },
            ],
          ]
        }>
        foo2 bar2
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Array [
              Object {
                "fontSize": 13.5,
              },
              Object {
                "color": "#000000",
                "fontSize": 18,
              },
            ],
          ]
        }>
        Há segundos
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#654C50",
              "fontFamily": "Avenir",
              "fontSize": 18,
            },
            Object {
              "color": "#807F7F",
              "fontSize": 15,
              "marginBottom": 6,
            },
          ]
        }>
        foo2 não descreveu sua experiência.
      </Text>
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginBottom": 4,
          }
        }>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
        <Text
          accessible={true}
          allowFontScaling={false}
          ellipsizeMode="tail"
          style={
            Array [
              Object {
                "color": "#654C50",
                "fontFamily": "Ionicons",
                "fontSize": 27,
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {
                "color": "#FFBA00",
              },
            ]
          }>
          
        </Text>
      </View>
    </View>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#654C50",
            "fontFamily": "Avenir",
            "fontSize": 18,
          },
          Array [
            Object {
              "color": "#654C50",
            },
            Array [
              Object {
                "marginBottom": 20,
                "marginTop": 10,
                "textAlign": "center",
              },
              Object {
                "marginTop": 20,
              },
            ],
          ],
        ]
      }>
      Carregar mais
    </Text>
  </View>
</View>
`;
